src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="container" data-qa="card">
+      <img
+        class="container__image"
+        src="./images/imac.jpeg"
+        alt="goods image"
+        >
+      <h2 class="container__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <p class="container__text">Код товара: 195434</p>
+
+      <div class="container__review">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <div class="container__review-count">
+          <p>Отзывов: 5</p>
+        </div>
+      </div>
+
+      <div class="container__price">
+        <p class="prices">Цена:</p>
+        <span class="suma">69 999 грн</span>
+      </div>
+
+      <div class="container__button">
+        <a data-qa="hover" href="#">
+          <button class="submit" type="submit">Купить</button>
+        </a>
+      </div>
```

Comment:
delete unnecessary wrappers and button - it's redundant.
```suggestion
        <a 
          data-qa="hover" 
          href="#"
          class="container__button"
        >
          Купить
        </a>
```